packages/manager-components/src/components/region-label/region-label.spec.tsx

```diff
@@ -0,0 +1,37 @@
+import { screen } from '@testing-library/react';
+import { render } from '../../utils/test.provider';
+import RegionLabel, { RegionLabelProps } from './region-label.component';
+import { DemoRegion, DemoDatacenter } from './region-label.stories';
+import translatedRegion from './translations/Messages_fr_FR.json';
+
+const renderComponent = (props: RegionLabelProps) => {
+  return render(<RegionLabel {...props} />);
+};
+
+describe('Region Label component', () => {
+  it('renders region label correctly for region key', async () => {
```

Comment:
```suggestion
  it('renders region label correctly for region key', async () => {
```
  it('renders region label correctly for region key', async () => {
    renderComponent({
      ...DemoRegion.args,
    });
    const regionKey = DemoRegion.args.name
      .replace(/-/g, '_')
      ?.toLocaleLowerCase();
    const regionLabelElement = screen.getByText(
      translatedRegion[`region_${regionKey}`],
    );
    expect(regionLabelElement).toBeVisible();
  });

  it('renders region label correctly for datacenter key', async () => {
    renderComponent({
      ...DemoDatacenter.args,
    });
    const regionKey = DemoDatacenter.args.name
      .replace(/-/g, '_')
      ?.toLocaleLowerCase();
    const regionLabelElement = screen.getByText('Roubaix (RBX2) - France');
    expect(regionLabelElement).toBeVisible();
  });